src/main/java/net/mcreator/blockly/java/ProcedureCodeOptimizer.java

```diff
@@ -127,11 +129,22 @@ else if (blacklist != null && parentheses == 1) {
 	}
 
 	/**
-	 * This method removes blockstate/itemstack markers from the given code
+	 * This method performs parentheses optimization and adds an (int) cast to the given code if needed.
+	 * @param code The code representing the number to cast
+	 * @return The code without parentheses, if it's already an int, or with a cast to (int) behind otherwise
+	 */
+	@SuppressWarnings("unused") public static String toInt(String code) {
+		if (code.startsWith("/*@int*/"))
+			return removeParentheses(code);

```

Comment:
Couldn't code here theoretically contain instanceof too, but there is no check for it here?

Reply:
There's no explicit cast in this case, so there's no need to check for lower priority operations